src/GafferImage/OpenImageIOReader.cpp

```diff
@@ -523,6 +523,19 @@ void OpenImageIOReader::hashFormat( const GafferImage::ImagePlug *output, const
 	hashFileName( context, h );
 	refreshCountPlug()->hash( h );
 	missingFrameModePlug()->hash( h );
+
```

Comment:
Why are you modifying `hashFormat` if you aren't modifying `computeFormat`? They should always match up I think.